### DISCUSSION TITLE
Logitech BRIO 4K Webcam won´t work after update

### DISCUSSION THREAD

**Discussion by find-usdk** [photobooth-app/photobooth-app#384] [Howtos and camera specific topics]
I have got the Logitech BRIO Webcam to work with Opencv2, but now after i have made upgrade with:
sudo apt-get update & sudo apt-get upgrade 
Are it not working any more, I have tried both Opencv2 and v4l2 with device index 0 and 1.

I have tried to run this command from terminal (Don´t know if its the right way):
python -c "from photobooth.services.backends.webcamv4l import *; print(available_camera_indexes())"

But i get this error:
![image](https://github.com/user-attachments/assets/0d65e093-e15a-4a54-b96d-a05ec088e694)

**Comment by mgineer85** [comment]
The error is about not finding PhotoBooth at all which means depending on your installation method you previously need to activate the venv.

once you can start the PhotoBooth again (maybe it’s running in the background still? Can you access the webpage?) it would be good to have logs to see what’s going on.

**Reply by mgineer85** [reply]
On Linux I would recommend v4l backend because it’s way less Ressource intensive.

**Comment by find-usdk** [comment]
I´m running it on a Rasperry Pi 5, and the webpage runs fine.
![image](https://github.com/user-attachments/assets/ed292690-ff55-4658-8908-1589999af453)


And have tried both Opencv2 and v4l2.

Attached are log:
[photobooth_20241223.log](https://github.com/user-attachments/files/18228181/photobooth_20241223.log)

**Reply by find-usdk** [reply]
I don’t know what happened, but now it suddenly works again, even with v4l.

I’m happy again, and all is good! 😊 Let the Christmas spirit begin!

Wish you a merry Christmas and happy new year.

**Reply by mgineer85** [reply]
good to know. I do not see an photobooth-app specific error in the logs anyways. 
Maybe it was due to system kernel upgrade and the webcamera became available only after restart?

Merry christmas :)
